### PR TITLE
Properly fix concurrency issue with QEMU cmd

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
Properly synchronize the QEMU output.
Instead of By in helper function use gingkowriter.
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/utils/pod.go [hunk 1]
```diff
@@ -146,7 +146,7 @@ func podStatus(clientSet *kubernetes.Clientset, podName, namespace string, statu
 			}
 			return false, err
 		}
-		ginkgo.By("Checking POD phase: " + string(pod.Status.Phase))
+		fmt.Fprintf(GinkgoWriter, "INFO: Checking POD phase: %s\n", string(pod.Status.Phase))
```
Comment (j-griffith) [inline tests/utils/pod.go]: We drop the dot import so you'd need "ginkgo.GingkoWriter" here (sorry, I know that sort of annoying), we could name the import if we want but no real value there IMO

Comment (awels) [reply tests/utils/pod.go]: No, that is good, I forgot to run make test-lint on this patch, and I should have.

**Comment by awels** [comment]
ci test please

**Comment by awels** [comment]
ci test please

**Comment by awels** [comment]
ci test please

**Pull request merged**